.goreleaser.yaml

```diff
@@ -0,0 +1,48 @@
+version: 2
+
+before:
+  hooks:
+    - go mod tidy
+
+builds:
+  - id: capo
+    binary: capo
+    dir: ./cmd/capo
+    ldflags:
+      - -extldflags "-static" -s -w -X main.commit={{.Commit}} -X main.date={{.Date}} -X main.builtBy=goreleaser -X main.Version={{.Version}} -X main.Revision={{.ShortCommit}}
```

Comment:
I guess this could have some explaining comment attached. All flags make sense except the first one